lms/static/js/discovery/collection.js

```diff
@@ -14,6 +14,7 @@ define([
         latestModelsCount: 0,
         searchTerm: '',
         facetList: {},
+        facets: {},
```

Comment:
It is somewhat confusing to have facetList and facets. Maybe we should rename the ones sent as search params to filters, or selectedFacets or something to make them easily distinguishable.